src/main/start-main-application/runnables/kube-config-sync/add-source.injectable.ts

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
As modern code like this emerges, it would be great if it did so accompanied by those darn behavioural tests ;) WDYT?

Reply:
Discussion handled IRL.